python.cmake

```diff
@@ -55,7 +55,11 @@ ENDIF(CMAKE_VERSION VERSION_LESS "3.2")
 
 MACRO(FINDPYTHON)
 
-FIND_PACKAGE(PythonInterp ${ARGN})
+IF(ARGN)
+  FIND_PACKAGE(PythonInterp ${ARGN})
+ELSE(ARGN)
+  FIND_PACKAGE(PythonInterp 3)
```

Comment:
Please do not amend default behaviour on released LTS distros. LTS distros will maintain their default (and thus 2.7) till the end of distro life.

Reply:
I did not think about that. I think @wxmerkt is correct.